src/lib/api/eventApi.ts

```diff
@@ -62,3 +62,27 @@ export async function createOptimalRouteText(event: Event): Promise<string> {
 
     return `Vår reiseplanlegger har beregnet en optimal rute der antall etapper er ${event?.optimalStepNumber} og reisetid er ${hours} timer, ${minutes} minutter og ${seconds} sekunder.`
 }
+
+export async function createNewEvent(
```

Comment:
Småpirk, men lettere å finne frem dersom funksjoner heter samme i backend og frontend, så enten:
createEvent i begge eller createNewEvent i begge.

Reply:
Takk for kommentar! Fiksa